velox/exec/Aggregate.h

```diff
@@ -208,6 +208,8 @@ class Aggregate {
   // 'result' and its parts are expected to be singly referenced. If
   // other threads or operators hold references that they would use
   // after 'result' has been updated by this, effects will be unpredictable.
+  // This method should be side-effect free, i.e., calling this method
+  // doesn't change the content of the accumnulators.
```

Comment:
accumnulators -> accumulators

It would be nice to explain why we have this requirement.